src/Conda.jl

```diff
@@ -369,4 +371,46 @@ function import_list(io::IO, args...; kwargs...)
     end
 end
 
+"""
+    pip_interop(bool::Bool, env::Environment=$ROOTENV)
+
+Sets the `pip_interop_enabled` value to bool.
+If `true` then the conda solver is allowed to interact with non-conda-installed python packages.
+"""
+function pip_interop(bool::Bool, env::Environment=ROOTENV)
+    runconda(`config --set pip_interop_enabled $bool --file $(conda_rc(env))`, env)
+end
+
+"""
+    pip_interop(env::Environment=$ROOTENV)
+
+Gets the `pip_interop_enabled` value from the conda config.
+"""
+function pip_interop(env::Environment=ROOTENV)
+    dict = parseconda(`config --get pip_interop_enabled --file $(conda_rc(env))`, env)["get"]
+    get(dict, "pip_interop_enabled", false)
+end
+
+function check_pip_interop(env::Environment=ROOTENV)
+    pip_interop(env) || error("""
+                              pip_interop is not enabled
+                              Use `Conda.pip_interop(true; [env::Environment=ROOTENV])` to enable
+                              """)
+end
+
+function _pip(env::Environment)
+    "pip" ∉ _installed_packages(env) && add("pip", env)
+    joinpath(bin_dir(env), "pip")
+end
+
+function pip(cmd::AbstractString, pkgs::PkgOrPkgs, env::Environment=ROOTENV)
+    check_pip_interop(env)
+    _cmd = String[split(cmd, " ")...]
+    @info("Running $(`pip $_cmd $pkgs`) in $(env==ROOTENV ? "root" : env) environment")
+    env_var = _get_conda_env(env)
+    
+    run(setenv(`$(_pip(env)) $_cmd $pkgs`, env_var))
```

Comment:
Why couldn't this use `_set_conda_env`?

Reply:
I've changed it to use `_set_conda_env`.  The `_pip(env)` command still needs to be given because unlike the base `conda`, the `pip` is run from the specified `env`